web3_proxy/src/bin/web3_proxy_cli/migrate_stats_to_v2.rs

```diff
@@ -169,6 +172,8 @@ impl MigrateStatsToV2 {
 
                     let request_ulid = Ulid::new();
 
+                    let latest_balance = Arc::new(AtomicF64::from(0.));
```

Comment:
lets just try it as a Decimal in a lock for now. If its a problem, we can come back to atomics

Reply:
ok, changing it now. Just for the record, it will be a Arc<RwLock<Decimal>>